libraries/botframework-connector/src/auth/microsoftAppCredentials.ts

```diff
@@ -28,14 +28,18 @@ export class MicrosoftAppCredentials implements msrest.ServiceClientCredentials
     public appPassword: string;
     public appId: string;
 
-    public oAuthEndpoint: string = Constants.ToChannelFromBotLoginUrl;
+    public oAuthEndpoint: string;
     public oAuthScope: string = Constants.ToChannelFromBotOAuthScope;
     public readonly tokenCacheKey: string;
     private refreshingToken: Promise<Response> | null = null;
 
-    constructor(appId: string, appPassword: string) {
+    constructor(appId: string, appPassword: string, channelAuthTenant: string = null) {
```

Comment:
Please replace this constructor with an overloaded constructor, rather than a null-able parameter. 

That allows for cleaner documentation and intellisense.

Reply:
👍

Reply:
Not exactly an overload. Chatted with Steve. Use nullable param instead.